unit-tests/dds/test-control-reply.py

```diff
@@ -36,7 +36,7 @@ def _on_control( server, id, control, reply ):
                 reply['sequence'] = n_replies            # to show that we've processed it
                 reply['nested-json'] = { 'more': True }  # to show off
                 return True  # otherwise the control will be flagged as error
-            server.on_control( _on_control )
+            subscription = server.on_control( _on_control )
```

Comment:
Looks like a fix that is not related to this PR

Reply:
Right. I don't know how it passed before.